docs/_config.yml

```diff
@@ -47,11 +47,11 @@ categories:
   Frameworks:
     en: Frameworks
     cn: 计算框架
-    pt: Estruturas
+    pt: Frameworks
```

Comment:
Yes you are right but as there is no "good" direct translation to Portuguese for 'Frameworks', it is best to leave it in English